templates/items/parts/item-activities.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="tab activities" data-group="primary" data-tab="activities">
+    {{#each system.activities}}
+    <div class="activity card" data-id="{{ _id }}">
+
+        {{!-- TODO: Add drag handle for re-ordering --}}
+
+        {{!-- Icon --}}
+        <div class="icon">
+            <dnd5e-icon src="{{ img }}"></dnd5e-icon>
```

Comment:
You can give activities custom icons, so we'll probably want to handle that similar to the advancement icons.